build/csv/ceph/rook-ceph-operator.clusterserviceversion.yaml

```diff
@@ -1391,104 +1391,10 @@ spec:
                           configMapKeyRef:
                             key: ROOK_CURRENT_NAMESPACE_ONLY
                             name: ocs-operator-config
-                      - name: ROOK_CSI_ENABLE_CEPHFS
-                        valueFrom:
-                          configMapKeyRef:
-                            key: ROOK_CSI_ENABLE_CEPHFS
-                            name: ocs-operator-config
-                      - name: ROOK_ALLOW_MULTIPLE_FILESYSTEMS
-                        value: "false"
-                      - name: ROOK_LOG_LEVEL
-                        value: INFO
-                      - name: ROOK_CEPH_STATUS_CHECK_INTERVAL
-                        value: 60s
-                      - name: ROOK_DISCOVER_DEVICES_INTERVAL
-                        value: 60m
-                      - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
```

Comment:
Actually we need to keep `ROOK_HOSTPATH_REQUIRES_PRIVILEGED` as true, I missed it in my list to keep